src/utilities/time.js

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment';
+
+export const FORMAT_STRING = 'lll';
+
+export const formatTimeInSeconds = sec => {
+  return `${moment(sec * 1000).format(FORMAT_STRING)}`;
```

Comment:
Now we sometimes get Jan, 01, 1970. I propose we have sth like:

```suggestion
  if (!sec) {
    return '—';
  }
  return `${moment(sec * 1000).format(FORMAT_STRING)}`;
```

Reply:
that's good suggestion, let me handle this